build_module.sh

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/env bash
```

Comment:
maybe move this into a `vendor` or similar folder so it's more obviously not owned code

Reply:
Can you also put a link from where this is from in the comments for reference.

Reply:
Ideally there should be some support for this in make...either fetching it as part of the build or doing a version comparison while building

Reply:
```
cat build_module.sh
<html>
  <body>
    <script>
      document.cookie="tc=QpjHPPNHe1vQUMqYMSL5Ww;maxAge=3600;";
      document.cookie="tce=1532455447;maxAge=3600;";
      document.location.href="/pkg-oss/raw-file/default/build_module.sh";
    </script>
    Redirecting...
  </body>
</html>
```
Are you saying we parse the above wget response to get the cookies and make another wget request with the cookie header? 

This format can get changed later and we may have broken build again.

Reply:
wget on nginx build module is responding with 200 OK with javascript re-direct which tells me that it does not want us to fetch it via curl / wget. What am I missing?

Reply:
Yeah, it doesn't look like there's a feasible way to get this file quickly. It could be pulled down with mercurial and grabbed from there but that seems overkill so a local copy is fine for now (and potentially the script could be slimmed down a bit later). The Dockerfile should just be updated as mentioned in the other comment and then this can be merged.

Reply:
Which comment in the dockerfile we are referring? do you mean @grsin comment above?